### ISSUE TITLE
sandbox empty after reset data and sync

### ISSUE THREAD

**Issue by librelois** [duniter/duniter#1027]
I apply `reset data` and `sync` command to my node `g1-monit.elois.org` and just after sync database sandbox is empty : https://librelois.fr/public/duniter-g1-33426.g1-monit.elois.org.db

I observe exactly the same behavior on my classic node `g1.elois.org`, so this behavior has no relationship with my module.

**Comment by librelois** [comment]
Sandbox crawling seem not work, node `g1-monit.elois.org` receive only six peending identities in 14 hours : https://librelois.fr/public/duniter-g1-33596.g1-monit.elois.org.db

**Comment by c-geek** [comment]
* `reset data` also resets the sandboxes, so that's normal this data disappear. At the end of the `sync`, the remote sandbox should be retrieved. But it might occur a timeout at that step, because the sandbox synchronization requires heavy load (#1014). In such case the sandbox sync is skept.
* since Duniter 1.3.0, sandbox crawling is done [once a day](https://github.com/duniter/duniter-crawler/blob/master/lib/constants.js#L19) (see https://github.com/duniter/duniter/issues/994). So you need to wait 24 hours to see your sandboxes full if the `sync` failed on that part. Or, you can replay `sync` command (withtout reset) to force the sandbox sync.

**Comment by c-geek** [comment]
I let this issue opened, but it is kind of a duplicate of #1014.

**Comment by M5oul** [comment]
I was asking me if a command exists to sync sandbox's data.

I found this one:
```bash
crawl-lookup <toHost> <toPort> [<fromHost] [<fromPort>]]          Make a full network scan and rebroadcast every WoT pending document (identity, certification, membership)
```

**Comment by c-geek** [comment]
Also linked to #1036

**Comment by c-geek** [comment]
Fixed by through #1036 and #1014.
